app/src/main/java/com/github/fribourgsdp/radio/game/prep/JoinGameActivity.kt

```diff
@@ -42,6 +46,8 @@ open class JoinGameActivity : MyAppCompatActivity() {
         joinButton = findViewById(R.id.joinGameButton)
         joinWithQRCodeButton = findViewById(R.id.joinWithQRCode)
         joinErrorView = findViewById(R.id.joinErrorView)
+        lobbiesRecyclerView = findViewById(R.id.publicLobbiesRecyclerView)
+        spinner = findViewById(R.id.lobbySortSpinner)
```

Comment:
If you have time you could do an initViews() as we have regularly done in other files, as well as an initBehaviours() or initListeners() for setting the listeners.

Reply:
Good idea 👍🏻

Reply:
Done in pr #242